### ISSUE TITLE
Automate error log notifications

### ISSUE THREAD

**Issue by NickSchimek** [codeforpdx/recordexpungPDX#640]
## Description
Currently logs are generated when there is an issue. We are notified by the client when an error occurs to check the logs.

It would be best to automate the notification process so that when an issue occurs we are notified automatically. 

Currently we have log files that are generated, however there are no automated ways to check these files. Heroku offers a task scheduler that we can setup to run once day that will check the log's for us. We can create a task to run daily that will check the log and notify us via Slack if there were any issues logged that day.

What do you think @codeforpdx/team-record-expungment ?

**Comment by KentShikama** [comment]
I don't think it will hurt to have. We haven't really been doing emergency hot fixes yet and we haven't really come across situations where we couldn't reproduce an issue with just a few parameters/names from Michael. But in the future this could become higher priority.

**Comment by alaymodi** [comment]
Hi, I'm new to the codebase but am interested in working on this project. Is this issue still open/unassigned

**Comment by KentShikama** [comment]
@alaymodi This issue won't really touch the codebase itself, and so I wouldn't recommend it as a first task. I would recommend https://github.com/codeforpdx/recordexpungPDX/issues/100 but tell us if you want to press on with this task regardless as it may require giving you some background on our production and Slack setup.

**Comment by alaymodi** [comment]
Sure! Ill check out that one first and will get back to you on this
